presto-main/src/test/java/com/facebook/presto/util/TestGraphvizPrinter.java

```diff
@@ -133,12 +133,12 @@ public void testPrintDistributedFromFragments()
         String expected = "digraph distributed_plan {\n" +
                 "subgraph cluster_0 {\n" +
                 "label = \"SOURCE\"\n" +
-                "plannode_1[label=\"{TableScan | [TableHandle \\{connectorId='connector_id', connectorHandle='com.facebook.presto.testing.TestingMetadata$TestingTableHandle@1af56f7', layout='Optional.empty'\\}]|Estimates: \\{rows: ? (0B), cpu: ?, memory: ?, network: ?\\}\n" +
+                "plannode_1[label=\"{TableScan | [TableHandle \\{connectorId='connector_id', connectorHandle='com.facebook.presto.testing.TestingMetadata$TestingTableHandle@1af56f7', layout='Optional.empty'\\}]|Estimates: \\{rows: ? (?), cpu: ?, memory: ?, network: ?\\}\n" +
                 "}\", style=\"rounded, filled\", shape=record, fillcolor=deepskyblue];\n" +
                 "}\n" +
                 "subgraph cluster_1 {\n" +
                 "label = \"SOURCE\"\n" +
-                "plannode_1[label=\"{TableScan | [TableHandle \\{connectorId='connector_id', connectorHandle='com.facebook.presto.testing.TestingMetadata$TestingTableHandle@1af56f7', layout='Optional.empty'\\}]|Estimates: \\{rows: ? (0B), cpu: ?, memory: ?, network: ?\\}\n" +
+                "plannode_1[label=\"{TableScan | [TableHandle \\{connectorId='connector_id', connectorHandle='com.facebook.presto.testing.TestingMetadata$TestingTableHandle@1af56f7', layout='Optional.empty'\\}]|Estimates: \\{rows: ? (?), cpu: ?, memory: ?, network: ?\\}\n" +
```

Comment:
Good change

Reply:
In this case, the size is 0B because the number of output variables is 0, i.e. no output at all, and outputing 0B sounds better here.